### PR TITLE
Project profile

### PR DESCRIPTION
**Pull request by gitcole**

### CODE REVIEW

**Comment by kmcnellis** [comment]
screen shot 2016-04-15 at 10 51 03 am

**Comment by kmcnellis** [comment]
It would be nice to make the contributors title larger, and add a "about" title to the description.

**Comment by kmcnellis** [comment]
The tech tags also don't look great. I preferred the little bubbles (like on the the developers page), but maybe there is something else you could do to show that its not just a long string of keywords?  Also, maybe put them under the description?

**Comment by kmcnellis** [comment]
screen shot 2016-04-15 at 10 52 42 am

**Comment by kmcnellis** [comment]
When editing, the upload images and x button should go under the picture.

File: client/app/user/user.html [hunk 1]
```diff
@@ -62,6 +55,24 @@
             </div>
         </div>
     </div>
+    <div class="row">
```
Comment (kmcnellis) [inline client/app/user/user.html]: Why are these edits to user.html in this pr?

File: server/api/classyear/classyear.controller.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ exports.index = function(req, res) {
   	"current": true
   }, function (err, classYear) {
   	if(err) { return handleError(res, err); }
-   	return res.json(200, classYear);
+   	return res.status(200).json(classYear);
```
Comment (kmcnellis) [inline server/api/classyear/classyear.controller.js]: Can these backend fixes be split into a different PR? They are unrelated (and causing conflicts)

**Comment by kmcnellis** [comment]
Also, the contributors should be sorted by name (or some metric)

**Comment by kmcnellis** [comment]
Marking the project as a past project is now missing (and should be available to admins/mentors without joining the project. Default project should only be available to admins & mentors (and without logging in)

File: client/app/user/user.html (hunk 1 above)
Comment (gitcole) [inline client/app/user/user.html]: Possibly because they got commited on the same branch

File: server/api/classyear/classyear.controller.js (hunk 2 above)
Comment (gitcole) [inline server/api/classyear/classyear.controller.js]: Depreciated function.  Didn't want to make a full PR just for a couple line changes

File: client/app/projects/profile/profile.controller.js [hunk 3]
```diff
@@ -182,7 +182,7 @@ angular.module('observatory3App')
     };
 
     $scope.checkUserProject = function() {
-        $scope.userOnProject = $scope.user.projects.indexOf($scope.project._id) !== -1;
+        $scope.userOnProject = $scope.user.projects.indexOf($scope.project._id) !== -1 || Auth.isMentor($scope.user);
```
Comment (kmcnellis) [inline client/app/projects/profile/profile.controller.js]: This doesn't work  - it shows the mentor as being on the project (shows the "leave project" buttons). It would be a separate check on the front end.

File: client/app/projects/profile/profile.controller.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ angular.module('observatory3App')
     $scope.imgPrefix = '/uploads/' + $stateParams.username + '/' + $stateParams.project + '/';
     $scope.userOnProject = false;
     $scope.project = {};
+    console.log(Auth);
```
Comment (kmcnellis) [inline client/app/projects/profile/profile.controller.js]: Remove console.log

**Comment by kmcnellis** [comment]
Mark past project and join/leave when a mentor still doesn't work, but the rest is looking good!

**Comment by gitcole** [comment]
Oh so people can't join if they are a mentor?

**Comment by kmcnellis** [comment]
No, its giving only the option to leave project

**Comment by kmcnellis** [comment]
Looks good, thanks!
